index.js

```diff
@@ -108,7 +116,7 @@ self.validate = function (params, schema) {
     });
 
     Object.keys(optional).forEach(function (optionalProperty) {
-        if (params[optionalProperty]) {
+        if (params[optionalProperty] !== undefined) {
```

Comment:
This avoids a valid, existing boolean being stripped off when it is set to false.